static/js/todo_widget.js

```diff
@@ -161,30 +161,64 @@ export function activate(opts) {
         const html = render_widgets_todo_widget();
         elem.html(html);
 
+        elem.find(".add-task-wrapper").hover(
+            (e) => {
+                e.stopPropagation();
+                check_valid_task();
+            },
+            (e) => {
+                e.stopPropagation();
+                check_valid_task();
+            },
+        );
```

Comment:
Can you add a comment explaining why this function is needed?

Reply:
Done!